Content/MsiInstallerExemptions.md

```diff
@@ -0,0 +1,15 @@
+# Msi Installer Exemptions
+
+_MsiInstallerExemptions_ is a sandbox setting in [Sandboxie Ini](SandboxieIni.md) available since v0.7.2 / 5.49.0.
+
+```
+   .
+   .
+   .
+   [DefaultBox]
+   MsiInstallerExemptions=y
+```
+
+Use the 'MsiInstallerExemptions=y' option to allow MSIServer to run with a sandboxed system token and apply other exceptions. This option may help with installing a msi package.
```

Comment:
```suggestion
Use the 'MsiInstallerExemptions=y' option to allow MSIServer to run with a sandboxed system token and apply other exceptions. This option may help with installing an MSI package.
```

The most common choice is to use "an MSI package" because "MSI" is typically pronounced as a single word.